components/usage/pkg/apiv1/usage_test.go

```diff
@@ -266,6 +272,7 @@ func TestReportGenerator_GenerateUsageReport(t *testing.T) {
 			UsageAttributionID: db.NewTeamAttributionID(teamID.String()),
 			CreationTime:       db.NewVarcharTime(time.Date(2022, 05, 1, 00, 00, 00, 00, time.UTC)),
 			StartedTime:        db.NewVarcharTime(time.Date(2022, 05, 1, 00, 00, 00, 00, time.UTC)),
+			StoppingTime:       db.NewVarcharTime(time.Date(2022, 06, 1, 1, 0, 0, 0, time.UTC)),
```

Comment:
It would be great to use slightly different times on for `StoppingTime` and `StoppedTime` as well as `CreationTime` `StartedTime` respectively. So the test better communicates that we measure between `StartedTime` and `StoppingTime`

Reply:
Good point, thanks! There is one test with real numbers from a DB example, but I guess the other tests could also have small differences between create/start and stopping/stopped.

Reply:
Yes, it's also easier to check as a human when using artificial numbers rather than real numbers.

Reply:
Fixed! Thanks again